paymentsheet/src/main/java/com/stripe/android/link/ui/updatecard/UpdateCardScreenState.kt

```diff
@@ -22,11 +23,8 @@ internal data class UpdateCardScreenState(
 
     val primaryButtonState: PrimaryButtonState
         get() = when {
-            cardModified.not() -> PrimaryButtonState.Disabled
+            cardModified.not() && isBillingDetailsUpdateFlow.not() -> PrimaryButtonState.Disabled
```

Comment:
Follow-up PR adding support for not requiring form modification to `CardDetailsUI`: https://github.com/stripe/stripe-android/pull/10997/files